stringfuzz/ast.py

```diff
@@ -5,16 +5,19 @@
 '''
 
 __all__ = [
-    'ASTNode',
```

Comment:
Is there any use for having things extend ASTNode if it is empty and other code can't check if things are an instance of it?

That is, should LiteralNode(ASTNode) just be LiteralNode(object) at this point?

Reply:
It exists as an abstract class for potentially adding functionality to all nodes, but I just wanted to make sure that it stays private.